legacyCore/src/main/res/values-sw300dp/themes.xml

```diff
@@ -0,0 +1,25 @@
+<resources>
+    <style name="Theme.Forzenbook" parent="Theme.Forzenbook.colorBase">
+        <!-- sizeing -->
+        <item name="smallPad_1">@dimen/paddingSmall1</item>
```

Comment:
These attributes don't do anything but wrap dimens that could just exist in `values-sw300dp` folders. Let's remove the dimen attributes unless you have a strong reason for needing them to be semantically named

Reply:
changed to dimens